frontend/src/Screen2.js

```diff
@@ -85,32 +62,36 @@ const MyForm = options => (
                         value={key}
                         label={options[key]}
                       />
-                    </CheckboxStyle>
+                    </Label>
                   )
                 })}
-              </div>
-            </div>
-            <LabelFormat>
-              <label htmlFor="whatWasInvolvedOther">
+              </Container>
+            </Container>
+            <H4>
```

Comment:
Small comment! Headers should be used for page structure only. In this case, since you're after larger font, you would want to use a paragraph with the styles applied to make the text appear how you want

Reply:
For sure, makes sense. I was just swapping out what was there but should have been thinking of this. Thanks!